test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -37,6 +37,7 @@ import (
 )
 
 const (
+	clusterSetUpTimeout   = 5 * time.Minute
```

Comment:
I'm not sure I like this name all that much. We use it in test set-up, but what we do is really just run some pods that we believe will fit on the existing nodes and wait for them to get scheduled. So I'd rather call it something like memoryReservationScheduleTimeout. I don't have a very strong opinion on this though, so if you think your name is better I'm ok to proceed with it.